scripts/generate_source.py

```diff
@@ -0,0 +1,110 @@
+#!/usr/bin/env python3
+# Copyright (c) 2023 Valve Corporation
+# Copyright (c) 2023 LunarG, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import argparse
+import os
+import sys
+from xml.etree import ElementTree
+
+def RunGenerators(api: str, registry: str, targetFilter: str) -> None:
+
+    # These live in the Vulkan-Docs repo, but are pulled in via the
+    # Vulkan-Headers/registry folder
+    # At runtime we inject python path to find these helper scripts
+    scripts = os.path.dirname(registry)
+    scripts_directory_path = os.path.dirname(os.path.abspath(__file__))
+    registry_headers_path = os.path.join(scripts_directory_path, scripts)
+    sys.path.insert(0, registry_headers_path)
+    from reg import Registry
+
+    from generators.base_generator import BaseGeneratorOptions
+    from generators.dispatch_table_helper_generator import DispatchTableHelperOutputGenerator
+    from generators.layer_dispatch_table_generator import LayerDispatchTableOutputGenerator
+    from generators.enum_string_helper_generator import EnumStringHelperOutputGenerator
+
+    # Build up a list of all generators and custom options
+    generators = {
+        'vk_dispatch_table_helper.h' : {
+            'generator' : DispatchTableHelperOutputGenerator,
+            'directory' : 'include/vulkan/layer',
```

Comment:
@juan-lunarg this is where we can simply change the location

Reply:
Gotcha 👍🏾

Reply:
@juan-lunarg  Do you know where you want these, then we can set them, run the script and check in the generated code

Reply:
Since these are all header only they should go in the `include/vulkan` directory.

Reply:
FYI @mikes-lunarg